### PR TITLE
{bio}[gompi/2019a] MetaBAT v2.14 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Darkless012** [comment]
Test report by @Darkless012
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3211.victini.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3d3eaf3bc92fcc8f9415c7d606b2c296 for a full test report.

File: easybuild/easyconfigs/m/MetaBAT/MetaBAT-2.14-gompi-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ('zlib', '1.2.11'),
+    ('SAMtools', '1.9'),
+    ('XZ', '5.2.4'),
+    ('bzip2', '1.0.6'),
+]
+
+prebuildopts = "cp -a $EBROOTSAMTOOLS samtools && "
+
+sanity_check_paths = {
+    'files': ['bin/aggregateBinDepths.pl', 'bin/aggregateContigOverlapsByBin.pl', 'bin/metabat', 'bin/metabat2'],
```
Comment (Micket) [inline easybuild/easyconfigs/m/MetaBAT/MetaBAT-2.14-gompi-2019a.eb]: We should have a Perl dependency here i think

Comment (boegel) [reply easybuild/easyconfigs/m/MetaBAT/MetaBAT-2.14-gompi-2019a.eb]: Good point, `Perl` should indeed be added as a dependency.

And we should make sure the right `perl` command is used by these scripts as well, by also using:

```python
fix_perl_shebang_for = ['bin/*.pl']

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/869d13f5cabbf397cfb0941ab7865e0d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2405.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/abb5723efa687aa4762134c46ee80d4f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3141.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5211d6d94bee253f892e348f48366098 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**